versioned_docs/version-1.3.0/install-kubeslice/kubeslice-cli/install-kubeslice.mdx

```diff
@@ -13,8 +13,27 @@ controller cluster and the worker cluster names. The file should include other r
 as described in the sample configuration file provided below. This file is essential in setting 
 up KubeSlice.
 
-Custom chart values can be specified in the topology configuration. For example, you can add `values:metrics.insecure=true` under 
+
+### Air-Gapped Installation
+
+To install KubeSlice in Air-Gapped environment use local helm charts. Upgrade to the latest 
+[kubeslice-cli version 0.6.0](https://github.com/kubeslice/kubeslice-cli/releases) to use local charts.
+
+Make sure you set the **use_local** value to `true` in the topology file:  
+
+```
+configuration:
+  helm_chart_configuration:
+    repo_alias: # {The alias of the helm repo for KubeSlice Charts. For local charts provide the local path to the charts.}
+    use_local: # {Use local charts instead of remote charts. Default is false}
+```
+
+### Custom Chart Values
+
+Custom chart values can be specified in the topology configuration, see [Topology Configuration Parameters](/versioned_docs/version-1.3.0/install-kubeslice/kubeslice-cli/topology-configuration.mdx).
+For example, you can add `values:metrics.insecure=true` under 
 **configuration.helm_chart_configuration.worker_chart** for worker charts.
```

Comment:
instead of bold format use `backticks` for property names/values.

Reply:
I had made the parameters in bold and the values in backticks